src/components/Composer.vue

```diff
@@ -835,6 +954,33 @@ export default {
 			this.state = STATES.DISCARDED
 			this.$emit('close')
 		},
+		/**
+		 * Whether the date is acceptable
+		 *
+		 * @param {Date} date The date to compare to
+		 * @returns {boolean}
+		 */
+		disabledDatetimepickerDate(date) {
+			const now = new Date()
+			const minimumDate = new Date(now.getTime())
```

Comment:
```suggestion
			const minimumDate = new Date()
```

the object already points to the current timestamp